bindings/rust/Cargo.toml

```
@@ -2,6 +2,10 @@
 name = "scip"
 version = "0.1.0"
 edition = "2021"
+license = "Apache-2.0"
+description = """
+SCIP is a protocol for communicating code intelligence information about projects.
```

```suggestion
SCIP (pronunciation: "skip") is a language-agnostic protocol for indexing source code, which can be used to power code navigation functionality such as Go to definition, Find references, and Find implementations.
```

(copied from our Readme)

---

smart